sonic-net/sonic-buildimage
# sonic-buildimage: Re-enable checking the pcie firmware is loaded in syncd for 720DT

#### Why I did it
Remove the check as the platform is strapped to load the pcie firmware and we should be able to keep the SDK check to ack the firmware was loaded instead of skipping the verification.

This is similar to https://github.com/sonic-net/sonic-buildimage/pull/13080 where we do not want to keep this check disabled if we do not need to.

#### How I did it
Removed the bcm config that enabled the skip behavior.

#### How to verify it
Ensure that the device is able to boot successfully and perform the check and not crash syncd.

#### Which release branch to backport (provide reason below if selected)

#### Tested branch (Please provide the tested image version)
